### PR TITLE
Expedition Fluff Expansion - New Survey Probes

### PR DESCRIPTION
**Pull request by 1Sparky1**
Splits the current survey probes into two types: Atmosphere and Ground, as well as adding a Geomagnetic variant. Replaces some of the currently mapped in atmosphere probes with the two other types.

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!review

File: code/modules/overmap/sectors.dm [hunk 1]
```diff
@@ -241,6 +251,27 @@ var/global/area/overmap/map_overmap
...
+	if(!wreck_details[WRECK_NAME])
+		wreck_details[WRECK_NAME] = name
+
+	if(!wreck_details[WRECK_AGE])
+		var/age = rand(1, max_age)
+		wreck_details[WRECK_AGE] = "Reconstruction of hull displacement suggests a high damage event occured approximately [age] years ago."
+
+	if(!wreck_details[WRECK_DAMAGE])
+		var/cause = pick("an equipment malfunction", "a meteor strike", "an electronic storm", "inter-vessel weaponry", "a collision")
+		wreck_details[WRECK_DAMAGE] = "Reconstruction indicates the damage was most likely cause by [cause]."
```
Comment (DreamySkrell) [inline code/modules/overmap/sectors.dm:273]: Honestly really not a fan of a hull probe like that. There's no way to figure out any of that information with a probe just sampling some metal.

This information should be up to actual players and their chars to figure out, by exploring the away site or ship.

Like how would the probe even know if it's sampling a wreck or not? At what point does a ship or station become a wreck? It sounds really arbitrary.

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
as above

**Comment by Triogenix** [comment]
Hi, having discussed this with my deputy, the LTA is also in agreement with what dreamy said above, and would like to see the hull probe removed from this PR before it is merged.

**Review by DreamySkrell — APPROVED** [review]

File: code/game/objects/structures/survey_probe.dm [hunk 2]
```diff
@@ -151,11 +114,33 @@
...
 
 		// fin
 		timer_id = null
-		icon_state = "surveying_probe_deployed"
+		icon_state = "[initial(icon_state)]_deployed"
+
+/obj/structure/survey_probe/proc/get_report(var/turf/T, var/is_exoplanet, var/is_asteroid)
+	. = "<b>Atmospheric survey results:</b>"
+	var/datum/gas_mixture/air = T.return_air()
+	if(air.total_moles>0)
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/survey_probe.dm:126]: this runtimes if there's no air

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**